### DISCUSSION TITLE
Welcome to Steal like an Rtist: Creative Coding in R! 🎨💻

### DISCUSSION THREAD

**Discussion by sharlagelfand** [posit-conf-2023/creative-coding#1] [Announcements]
Share your name and favorite and least favorite colors here!

**Comment by sshaughnessy** [comment]
schaefer shaughnessy
favorite color - blue
least favorite - orange

**Comment by natalieoshea** [comment]
Hi, I'm Natalie! My favorite color is periwinkle blue that's a little more on the lavender side and my least favorite color is construction-cone orange

**Comment by Meghansaha** [comment]
I'm Meghan. 

- My favorite Color is "Blurple" - Black, Blue, and Purple, standard colors of a space/nebula 🌌
- My least favorite color Orange   🟠

**Comment by bshk93** [comment]
Name: Brian
Favorite color: Forest green/Olive 
Least favorite color: Lime green

**Comment by thomaspara** [comment]
Hi, I'm Thomas Parashos, my favorite is #59E5FB, my least favorite is #FFFFFF, too bright

**Reply by thomaspara** [reply]
since peeps asked :)
keyboard:
https://www.gloriousgaming.com/products/glorious-gmmk-pro-75-barebone-black
https://www.gloriousgaming.com/products/gmmk-pro-top-frame
https://www.gloriousgaming.com/products/gmmk-pro-75-switch-plate?variant=38119675625647
https://drop.com/buy/everglide-aqua-king-mechanical-switches?defaultSelectionIds=980106
https://drop.com/buy/drop-dsa-astrolokeys-keycaps-by-sailorhg-and-cassidoo?searchId=50368702db524a0df0baf7d7f6e4f784&defaultSelectionIds=966965
https://www.adafruit.com/product/5531
macropad:
https://www.adafruit.com/product/5128
https://vala.supply/products/dsa-petrikeys
https://www.adafruit.com/product/5529

**Comment by roBertus31** [comment]
Rob Teal
Most: Cerulean
Least: Beige

**Comment by iamciera** [comment]
As was already said:

Favorite color Blue.
Least favorite Orange

**Comment by lwdozal** [comment]
Laura
mint
highlight yellow

**Comment by Kim-s-h** [comment]
Hi I'm Sonia, my favorite color is dark navy, my least favorite is purple.

**Comment by shaunmwilkinson** [comment]
Shaun Wilkinson
Most: Green
Least: Yellow-Green

**Comment by dougtommet** [comment]
Doug Tommet
Favorite color: purple
Least favorite color: beige

**Comment by datatoolbox** [comment]
David D - earth tones ...browns and greens

**Comment by MHanken** [comment]
Marlene - LOVE--all colors especially peacock array!  Not a fan of Brown.

**Comment by maxrogow** [comment]
Max--the deep blue on christmas lights

**Comment by datatoolbox** [comment]
![ddaniel](https://github.com/posit-conf-2023/creative-coding/assets/5366795/ba44353d-a7b9-4433-a3b8-bfbe943c7733)
